src/components/common/Extensions/UserHubButton/UserHubButton.tsx

```diff
@@ -151,7 +151,7 @@ const UserHubButton: FC = () => {
           {
             '!border-blue-400': visible || isUserHubOpen,
           },
-          'md:hover:!border-blue-400',
+          ' flex min-w-[3rem] md:hover:!border-blue-400',
```

Comment:
I'm being incredibly nitpicky, but there is an unnecessary space at the start of this line.

Reply:
I also don't think flex is needed here as it is already included in the button component.